vizalerts.py

```diff
@@ -195,6 +195,7 @@ def process_views(views):
         sitename = unicode(view["site_name"]).replace('Default', '')
         viewurlsuffix = view['view_url_suffix']
         viewname = unicode(view['view_name'])
+        logger.debug(viewurlsuffix)
```

Comment:
It was left in by accident, it can be removed.

Jonathan


On Tue, Mar 1, 2016 at 4:02 PM, Matt <notifications@github.com> wrote:

> In vizalerts.py
> <https://github.com/tableau/VizAlerts/pull/14#discussion_r54635057>:
>
> > @@ -195,6 +195,7 @@ def process_views(views):
> >          sitename = unicode(view["site_name"]).replace('Default', '')
> >          viewurlsuffix = view['view_url_suffix']
> >          viewname = unicode(view['view_name'])
> > +        logger.debug(viewurlsuffix)
>
> Was this line left in by accident? We can definitely add the full string
> in to the log, but I'd prefer we moved it up to the call to logger.debug a
> few lines up, so we can keep the info in the one line of log text.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tableau/VizAlerts/pull/14/files#r54635057>.
>